environment.yml

```diff
@@ -1,8 +1,8 @@
-name: taxcalc-dev
+name: taxcalc-india-dev
```

Comment:
@kumar-ab You changed the name of the environment here.  So you'll need to change it in `.travis.yml` in the line
```
source activate taxcalc-dev
```
That might fix the latest error for you.